src/sentry/preprod/tasks.py

```diff
@@ -330,3 +330,64 @@ def assemble_preprod_artifact_size_analysis(
             assemble_result, project, artifact_id, org_id
         ),
     )
+
+
+def _assemble_preprod_artifact_installable_app(
+    assemble_result: AssembleResult, project, artifact_id, org_id
+):
+    from sentry.preprod.models import PreprodArtifact
```

Comment:
can we move this import to the top so that all the different methods can share it?